src/EFCore.PG/Query/ExpressionTranslators/Internal/NpgsqlFullTextSearchMethodTranslator.cs

```diff
@@ -107,75 +108,61 @@ static Expression TryTranslateOperator(MethodCallExpression e)
             }
         }
 
-        static Expression TryTranslateFunction(MethodCallExpression methodCallExpression)
+        [CanBeNull]
+        static Expression TryTranslateFunction([NotNull] MethodCallExpression e)
         {
-            switch (methodCallExpression.Method.Name)
+            switch (e.Method.Name)
             {
             case nameof(NpgsqlFullTextSearchLinqExtensions.GetNodeCount):
-                return new SqlFunctionExpression(
-                    "numnode",
-                    methodCallExpression.Method.ReturnType,
-                    methodCallExpression.Arguments);
+                return new SqlFunctionExpression("numnode", e.Method.ReturnType, e.Arguments);
 
             case nameof(NpgsqlFullTextSearchLinqExtensions.GetQueryTree):
-                return new SqlFunctionExpression(
-                    "querytree",
-                    methodCallExpression.Method.ReturnType,
-                    methodCallExpression.Arguments);
+                return new SqlFunctionExpression("querytree", e.Method.ReturnType, e.Arguments);
 
             case nameof(NpgsqlFullTextSearchLinqExtensions.GetResultHeadline):
-                var tsheadlineFunctionName = "ts_headline";
-                switch (methodCallExpression.Arguments.Count)
+                const string tsheadlineFunctionName = "ts_headline";
```

Comment:
I'd just put this inline, clearer/easier to read and no value in having it extracted as a const (the function name isn't going to change).